src/libunicode/emoji_segmenter.h

```diff
@@ -140,66 +137,64 @@ inline std::ostream& operator<<(std::ostream& os, EmojiSegmentationCategory valu
         case unicode::EmojiSegmentationCategory::RegionalIndicator: return os << "RegionalIndicator";
         case unicode::EmojiSegmentationCategory::KeyCapBase: return os << "KeyCapBase";
         case unicode::EmojiSegmentationCategory::CombiningEnclosingKeyCap: return os << "CombiningEnclosingKeyCap";
-        case unicode::EmojiSegmentationCategory::CombiningEnclosingCircleBackslash: return os << "CombiningEnclosingCircleBackslash";
+        case unicode::EmojiSegmentationCategory::CombiningEnclosingCircleBackslash:
+            return os << "CombiningEnclosingCircleBackslash";
         case unicode::EmojiSegmentationCategory::ZWJ: return os << "ZWJ";
         case unicode::EmojiSegmentationCategory::VS15: return os << "VS15";
         case unicode::EmojiSegmentationCategory::VS16: return os << "VS16";
         case unicode::EmojiSegmentationCategory::TagBase: return os << "TagBase";
         case unicode::EmojiSegmentationCategory::TagSequence: return os << "TagSequence";
         case unicode::EmojiSegmentationCategory::TagTerm: return os << "TagTerm";
-        // clang-format off
     }
     return os;
 }
 
 } // namespace unicode
 
-// clang-format off
-#if __has_include(<fmt/ostream.h>)
-#include <fmt/ostream.h>
-#if FMT_VERSION >= (9 * 10000 + 1 * 100 + 0)
-template <> struct fmt::formatter<unicode::PresentationStyle>: fmt::ostream_formatter {};
-
-
-namespace fmt
-{
 template <>
-struct formatter<unicode::EmojiSegmentationCategory>
+struct std::formatter<unicode::PresentationStyle>: std::formatter<std::string_view>
 {
-    template <typename ParseContext>
-    constexpr auto parse(ParseContext& ctx)
+    template <typename FormatContext>
+    auto format(unicode::PresentationStyle value, FormatContext& ctx) const -> FormatContext::iterator
     {
-        return ctx.begin();
+        string_view name;
+        switch (value)
+        {
+            case unicode::PresentationStyle::Text: name = "Text"; break;
+            case unicode::PresentationStyle::Emoji: name = "Emoji"; break;
+        }
+        return formatter<string_view>::format(name, ctx);
     }
+};
+
+template <>
+struct std::formatter<unicode::EmojiSegmentationCategory>: std::formatter<std::string_view>
+{
     template <typename FormatContext>
-    auto format(unicode::EmojiSegmentationCategory value, FormatContext& ctx)
+    auto format(unicode::EmojiSegmentationCategory value, FormatContext& ctx) const -> FormatContext::iterator
```

Comment:
I think we can make context to be auto since we do not need it, and omit template as well 
```suggestion
    auto format(unicode::EmojiSegmentationCategory value, auto& ctx) const 
```